style.css

```diff
@@ -1 +1,217 @@
-/*  Add your styling here */
+/* General body */
+
+body{
+    background-color: rgb(255, 255, 255);
+    font-family: Arial !important;
```

Comment:
Never use `!important`!
If you feel that you need it, you are probably trying to override a rule to a selector somewhere that is more 'specific' than `body`
[More details here ](https://dev.to/emmabostian/css-specificity-1kca)